backend.c

```diff
@@ -2485,6 +2485,7 @@ static void run_threads(struct sk_out *sk_out)
 				} else if (__td_index == fio_debug_jobno)
 					*fio_debug_jobp = pid;
 				free(eo);
+				eo = NULL;
```

Comment:
I don't see how this can fix anything - you're just clearing 'eo' after it has been freed.